SonarQube.Analysis.xml

```diff
@@ -1,34 +0,0 @@
-<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Should this file be removed? I'm unfamiliar with the setup of this project but I noticed that one of the CIs failed.

Reply:
It's a left-over file from when I was testing out SonarQube, a code quality tool. It's not in use anymore. I too saw that only one build failed; the test coverage tool doesn't always play nice with dotnet core, so once in a while it fails, but I cannot figure out why.